test/units/testsuite-73.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+set -eux
+set -o pipefail
+
+setup_cron() {
+    # Setup test user and cron
+    useradd test
+    echo "test" | passwd --stdin test
```

Comment:
Is it necessary to set password for the user?

Note, IIRC, passwd in Ubuntu CI does not support `--stdin`.

Reply:
Setting password shouldn't be necessary.